README.md

```diff
@@ -17,6 +17,28 @@ To get started with this sample, you have two options:
   [Quickstart: Add sign-in with Microsoft to a Python web app](https://docs.microsoft.com/azure/active-directory/develop/web-app-quickstart?pivots=devlang-python).
 * Use PowerShell scripts that automatically create the Azure AD applications and related objects (passwords, permissions, dependencies) for you, and then modify the configuration files. Follow the steps in the [App Creation Scripts README](./AppCreationScripts/AppCreationScripts.md).
 
+# Deployment
+
+Once you finish testing this web app locally, you can deploy it to your production.
+You may choose any web app hosting services you want.
+Here we will describe how to deploy it to
+[Azure App Service](https://azure.microsoft.com/en-us/products/app-service).
+
+* Follow the ["Quickstart: Deploy a Python (Django or Flask) web app to Azure App Service"](https://learn.microsoft.com/en-us/azure/app-service/quickstart-python),
+  but replace its sample app (which does not do user sign-in) with this web app.
+
+* In particular, if you choose to ["deploy using Local Git" in "step 3 - Deploy your application code to Azure"](https://learn.microsoft.com/en-us/azure/app-service/quickstart-python?tabs=flask%2Cwindows%2Cazure-cli%2Clocal-git-deploy%2Cdeploy-instructions-azportal%2Cterminal-bash%2Cdeploy-instructions-zip-azcli#3---deploy-your-application-code-to-azure),
+  an [application-scope credential](https://learn.microsoft.com/en-us/azure/app-service/deploy-configure-credentials?tabs=portal#appscope)
+  will be automatically created with the shape as `your_app_name\$your_app_name`.
+  But your actual git username is only the `$your_app_name` part.

```

Comment:
Is that a bug with the tutorial? Or somehow unique to this application? (If a bug with the tutorial, we should fix it there)

Reply:
It is not a bug in tutorial. In fact, the tutorial I quoted already gives a hint albeit buried deep inside the instruction.
But it has been an FAQ [here](https://github.com/MicrosoftDocs/azure-docs/issues/6548) and [there](https://learn.microsoft.com/en-us/answers/questions/870466/local-git-authentication-fail?page=1&orderby=helpful). And it can be said as a bug in the Portal UI. I reported it, and CCed you in the mail thread. Unless it is fixed in Portal, I suggest we leave this line here. It will save each future person a hour or two of frustrating fiddling.

Reply:
Okay, I need to give it a go myself to wrap my head around the bug as I don't remember it, thanks for raising to the team. Leaving the extra note here is fine.

Reply:
Just did the flow, I see what you mean. There's another issue with the flow in that the Quickstart presumes the branch is called "main" locally but this repo has it called "master". Can you rename it? I did it for my old repo last week and GitHub makes it pretty easy to rename in the settings. Most samples use "main" that I've seen.